tests/BaseArrayCollectionTestCase.php

```diff
@@ -20,7 +20,7 @@
 use function next;
 use function reset;
 
-abstract class BaseArrayCollectionTest extends TestCase
+abstract class BaseArrayCollectionTestCase extends TestCase
```

Comment:
```suggestion
abstract class ArrayCollectionTestCase extends TestCase
```

The `Base` prefix feels redundant now.